sw/device/silicon_creator/manuf/extensions/BUILD.bazel

```
@@ -29,6 +29,8 @@ cc_library(
         "@//sw/device/lib/testing/test_framework:status",
         "@//sw/device/lib/testing/test_framework:ujson_ottf",
         "@//sw/device/silicon_creator/lib/cert",
+        "@//sw/device/silicon_creator/manuf/base:ft_personalize_base",
+        "@//sw/device/silicon_creator/manuf/lib:flash_info_fields",
```

After making the new cc_lib I mention above, then do:
```suggestion
        "@//sw/device/silicon_creator/manuf/base:personalize_ext",
```

---

I thought I included flash_info_fields for a reason, but seems to be not necessary...

done